com.playeveryware.eos/Runtime/Core/EOSManager.cs

```diff
@@ -533,10 +533,12 @@ private PlatformInterface CreatePlatformInterface()
             //-------------------------------------------------------------------------
             private void InitializeOverlay(IEOSCoroutineOwner coroutineOwner)
             {
+                EOSConfig configData = Config.Get<EOSConfig>();
+
                 // Sets the button for the bringing up the overlay
                 var friendToggle = new SetToggleFriendsButtonOptions
```

Comment:
Instead of using the nomenclature of "activating the overlay," since that could mean more than one thing - does it make sense to change the language (such as the name of the field member added to `EOSConfig`) to something more specific like, `toggleFriendsInput`?